src/components/UpgradeTableOfContents.tsx

```diff
@@ -35,12 +32,6 @@ const TableOfContentsLink: React.FC<IPropsTableOfContentsLink> = ({ item: { titl
   )
 }
 
-interface IPropsItemsList {
-  items: Array<ToCItem>
-  depth: number
-  maxDepth: number
-}
-
 const ItemsList: React.FC<IPropsItemsList> = ({ items, depth, maxDepth }) => {
   if (depth > maxDepth || !items) return null
```

Comment:
we could add a variable name explaining what this conditional means and use that inside the `if` instead, as its hard to follow what's happening

`x = depth > maxDepth || !items`, where `x` should be a declarative name explaining the condition

Reply:
Okay.. I had trimmed the tertiary statement reflexively here, but careful of scope creep here (the logic here isn't new)... Similar to above, note that we had discussed not letting the relocation of all of these utils be a blocker for this migration.

Reply:
Not talking about a relocation but extracting this conditional to a declarative variable to make code easier to follow, it's not trivial to understand what's happening without context. It'd help to add some comments too